### ISSUE TITLE
About reproducing

### ISSUE THREAD

**Issue by liujikun** [universome/alis#11]
Hi,

Thank you for your great work! Have you ever cleaned the lhq dataset? I used the lhq_1024_jpg dataset to reproduce the effect, and the FID can only reach 9. I tried to fine-tune on the open source model, but the initial FID was as high as 24. 

Best,
JiKun

**Comment by universome** [comment]
Hi! I am sorry for answering that late. Yes, we indeed preprocessed the dataset (as specified in the Experiments section) with the procedure described in Section 3.3/Appendix C/Algorithm 1.

[Here](https://gist.github.com/universome/3140f74058a48aa56a556b0d9e24e857) is the script that we used to preprocess the datasets (we use a threshold of 0.95 for LHQ).

Also note, that in our Table 1, we provide results for the 256x256 resolution, not for 1024x1024 (for 1024x1024, our model achieved FID/∞-FID of 10.11/10.53). 

That's strange that the initial FID was so high. But maybe those "unconnectable" images (that are removed by our preprocessing procedure) really change the distribution so much.

**Comment by universome** [comment]
Hmm, the currently released dataset has images sorted by their InceptionV3 likelihood (from least to most probable). I think in the above script, one should also shuffle the images. I will update it

**Comment by Muccul** [comment]
@universome Hi, some confusion about your [jupyter](https://gist.github.com/universome/3140f74058a48aa56a556b0d9e24e857)  of preprocessing. 
LHQ256 dataset contain 90k data, and rest contain 70k after split(15k train, 5k val), but in your jupyer file, why filtering number is 167k with 0.99 threshold?
![image](https://user-images.githubusercontent.com/34998097/159107182-faa653ab-8afc-43fd-87a7-4343acff6e31.png)

**Comment by Muccul** [comment]
> @universome Hi, some confusion about your [jupyter](https://gist.github.com/universome/3140f74058a48aa56a556b0d9e24e857) of preprocessing. LHQ256 dataset contain 90k data, and rest contain 70k after split(15k train, 5k val), but in your jupyer file, why filtering number is 167k with 0.99 threshold? ![image](https://user-images.githubusercontent.com/34998097/159107182-faa653ab-8afc-43fd-87a7-4343acff6e31.png)

btw, if we get a subset of data, How to divide it into training set and test set ? and calculate fid just on test set in subset for your paper result?
Looking forward to your reply, thank you!
